rts/at/tx/src/test/java/org/jboss/jbossts/star/test/BaseTest.java

```diff
@@ -67,8 +66,7 @@ public class BaseTest {
 
     protected static final ExecutorService executor = Executors.newFixedThreadPool(4);
     protected static boolean USE_NETTY = false;
-    protected static boolean USE_UNDERTOW = false;
-    protected static boolean USE_QUARKUS = true;
+    protected static boolean USE_UNDERTOW = true;
```

Comment:
This is definitely an option for me to move from Quarkus and Undertow to raw Undertow, but if I understood correctly having the Quarkus used helped to test some Quarkus annotation or something? @mmusgrov please can you help share a bit more context on the benefit to running this test with Quarkus?

Reply:
We, Sweta and I, added it because the work was to do with SRA annotations, aka annotations for REST-AT, and that work was designed to facilitate the introduction of a quarkus rest-at extension and having the tests runs on quarkus, the target platform, was the correct thing to do. Such an extension with annotations will give users alternatives to WS-AT.

Reply:
OK, so we can try to run this test twice then, once with Undertow (on JDK 11 and above) and once with Quarkus (when using JDK 17 and above as Quarkus is not supported on JDK 11 if I understood correctly).

Reply:
@marcosgopen please can you raise a JBTM to re-add Quarkus testing of this class for SRA annotation testing on JDK 17 and above and to have the Quarkus dependency updated to a supported one? We can't stay on 3.2.6, 3.8 seems to be the oldest LTS version (https://quarkus.io/security/) and JDK 17 is the minimum since 3.7 (https://quarkus.io/blog/java-17/).

Reply:
Created, thanks. https://issues.redhat.com/browse/JBTM-3943